### PR TITLE
[FIX] web : display currency in correct place for rtl lang

### PR DESCRIPTION
**Pull request by Mtaylorr**
This fix is a continuation of an old fix https://github.com/odoo/odoo/pull/121689 After fixing the display of numbers to be always on the right , The symbol also should be display the same as in english from ltr

Steps to reproduce the issue :
1-install arabic language
2-go to accounting / customer invoices and you can see the placement of the symbol is reversed

before : 
before

after : 
after

opw-3295573

### CODE REVIEW

File: addons/web/static/tests/views/list_view_tests.js [hunk 1]
```diff
@@ -261,9 +261,9 @@ QUnit.module("Views", (hooks) => {
         assert.containsN(target, "tbody tr", 4, "should have 4 rows");
         assert.containsOnce(target, "th.o_column_sortable", "should have 1 sortable column");
 
-        assert.containsOnce(
-            target,
-            "thead th:nth(2) .text-end",
+        assert.strictEqual(
+            $(target).find("thead th:nth(2)").css("text-align"),
+            "right",
```
Comment (hubvd) [inline addons/web/static/tests/views/list_view_tests.js:266]: This doesn't do the same thing as before. You can simply change the class used.

**Review by hubvd — COMMENTED** [review]
Seems good, just one comment on the test you edited.
Also, this probably needs a test in `list_view_tests.js`

**Comment by Mtaylorr** [comment]
Hey @aab-odoo , @BastienFafchamps can you take a look at this FIX ? thanks

File: addons/web/static/src/views/list/list_renderer.js [hunk 2]
```diff
@@ -31,6 +32,7 @@ import {
     useState,
     useEffect,
 } from "@odoo/owl";
+
```
Comment (aab-odoo) [inline addons/web/static/src/views/list/list_renderer.js:35]: not necessary, but the below import could be moved up

File: addons/web/static/tests/views/list_view_tests.js [hunk 3]
```diff
@@ -17337,4 +17337,41 @@ QUnit.module("Views", (hooks) => {
...
+        });
+
+        await makeView({
+            type: "list",
+            resModel: "foo",
+            serverData,
+            arch: '<tree><field name="foo"/><field name="amount_currency"/></tree>',
+        });
+
+        console.log($(target));
```
Comment (aab-odoo) [inline addons/web/static/tests/views/list_view_tests.js:17352]: remove :-)

File: addons/web/static/tests/views/list_view_tests.js [hunk 4]
```diff
@@ -17337,4 +17337,41 @@ QUnit.module("Views", (hooks) => {
         assert.strictEqual(input, document.activeElement);
         assert.strictEqual(input.value, 'Value 1');
     });
+    QUnit.test("monetary field display for rtl langauges", async function (assert){
```
Comment (aab-odoo) [inline addons/web/static/tests/views/list_view_tests.js:17340]: languages

File: addons/web/static/tests/views/list_view_tests.js [hunk 5]
```diff
@@ -17337,4 +17337,41 @@ QUnit.module("Views", (hooks) => {
...
+            "right",
+            "Monetary cells should be right alined"
+        );
+
+        assert.strictEqual(
+            $(target).find("tbody tr:first td:nth(2)").css("direction"),
+            "ltr",
+            "Monetary cells should have ltr direction"
+        );
+
```
Comment (aab-odoo) [inline addons/web/static/tests/views/list_view_tests.js:17375]: not necessary blank line

**Review by aab-odoo — APPROVED** [review]
@hubvd did you test both ltr and rtl? I agree with the fix, but I haven't tested myself.

**Comment by Mtaylorr** [comment]
I think the only problem here is that if the xml view isn't updated , it won't work correctly

**Comment by hubvd** [comment]
> did you test both ltr and rtl? 

Yup, it works in both cases.

> if the xml view isn't updated

If you're talking about `list_renderer.xml`, this isn't a view but a template and as such, it will be updated automatically :)

**Comment by Mtaylorr** [comment]
@BastienFafchamps can you take a look at this FIX ?

**Comment by BastienFafchamps** [comment]
@robodoo r+

**Pull request closed without merging**